### PR TITLE
Add my project

### PR DESCRIPTION
**Pull request by VadimBezhkov**
Bulk-Thumbnail-Creator

### CODE REVIEW

**Comment by VadimBezhkov** [comment]
See pls

File: Bezhkov/Bulk-Thumbnail-Creator/ConsoleApp14/Images.cs [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+        static int count = 0;
+
+        internal static string path { get; set; }
+        internal static string path2 { get; set; }
+        public string Name { get; set; }
+
+        //Checking and setting the image size
+        internal void ResizeParametrs()
+        {
+            Console.WriteLine("Enter width");
```
Comment (Audyman) [inline Bezhkov/Bulk-Thumbnail-Creator/ConsoleApp14/Images.cs:25]: remove Console references from this class - move to Program.cs

File: Bezhkov/Bulk-Thumbnail-Creator/ConsoleApp14/Images.cs [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+            int.TryParse(Console.ReadLine(), out width);
+
+            Console.WriteLine("Enter height");
```
Comment (Audyman) [inline Bezhkov/Bulk-Thumbnail-Creator/ConsoleApp14/Images.cs:28]: remove Console references from this class - move to Program.cs

File: Bezhkov/Bulk-Thumbnail-Creator/ConsoleApp14/Images.cs [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+            string[] files = Directory.GetFiles(path);
+
+            foreach (string image in files)
+            {
+                if (token.IsCancellationRequested)
+                {
+                    Console.Clear();
+                    Console.WriteLine(new string('-', 119));
+                    Console.WriteLine("Task canceled");
+                    Console.WriteLine(new string('-', 119));
```
Comment (Audyman) [inline Bezhkov/Bulk-Thumbnail-Creator/ConsoleApp14/Images.cs:61]: remove Console references from this class - move to Program.cs

File: Bezhkov/Bulk-Thumbnail-Creator/ConsoleApp14/ManagerProgramm.cs [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+using System.Drawing;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace ConsoleApp14
+{
+    internal class ManagerProgramm
```
Comment (Audyman) [inline Bezhkov/Bulk-Thumbnail-Creator/ConsoleApp14/ManagerProgramm.cs:12]: Better to move this code to Program.cs as private methods

File: Bezhkov/Bulk-Thumbnail-Creator/ConsoleApp14/ManagerProgramm.cs [hunk 5]
```diff
@@ -0,0 +1,68 @@
...
+            Console.WriteLine();
+            Console.ResetColor();
+        }
+
+        // Text menu info
+        internal void ActionMenu()
+        {
+            Console.WriteLine();
+            Console.ForegroundColor = ConsoleColor.Magenta;
+            Console.WriteLine("Press enter 1 - Resize all image");
```
Comment (Audyman) [inline Bezhkov/Bulk-Thumbnail-Creator/ConsoleApp14/ManagerProgramm.cs:53]: Better to use separate class for storing those constants/messages (e.g. Constants as was shown on the previous lesson)

**Review by Audyman — CHANGES_REQUESTED** [review]

**Pull request closed without merging**